Screenbox/Pages/AllVideosPage.xaml

```diff
@@ -60,15 +61,15 @@
         <muxc:ProgressBar
             x:Name="LoadingIndicator"
             Margin="0,8,0,0"
-            Padding="{StaticResource ContentPageRightMargin}"
+            Padding="{StaticResource PageContentMargin}"
```

Comment:
Having both left and right padding is broken for `ProgressBar`.

https://github.com/huynhsontung/Screenbox/assets/31434093/62329210-f4c4-4d2b-a1a9-3e699529895a

Reply:
> Having both left and right padding is broken for `ProgressBar`.
> 
>  Recording.2023-10-11.220005.mp4

Another big miss, how do I fix this? Just tweak the margins like the breadcrumb bars?